lib/js/atoms/SurveyQuestion/SurveyQuestionScale.vue

```diff
@@ -0,0 +1,245 @@
+		margin-left: $space-xxxxs;
+	}
+
+	&__content {
+		background: $color-background-secondary;
+		border-radius: $radius-s;
+		display: flex;
+		justify-content: space-between;
+		padding: $space-s $space-xxs;
+		overflow-x: auto;
+
+		@media #{breakpoint-s()} {
+			padding: $space-s $space-l;
+		}
+	}
+
+	&__toggle {
+		display: flex;
+		justify-content: center;
+		margin-right: $space-xxs;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+
+	&__toggleSeparator {
+		display: none;
+
+		@media #{breakpoint-s()} {
+			width: $space-l;
+			display: block;
+		}
+	}
+
+	&__separator {
+		border-bottom: none;
+		border-top: 1px solid $color-mischka-gray;
+		margin: $space-m 0 $space-s 0;
+	}
+
+	&__elaboration {
+		display: flex;
+		flex-direction: column;
+	}
+
+	&__elaborationLabel {
+		@include headlineXS();
+	}
+
+	&__elaborationInput {
+		margin: $space-xxs 0;
+	}
+}
+</style>
+
+<script lang="ts">
+import { Prop } from 'vue/types/options';
+
+import DsCard from '../Card';
+import IconButton from '../IconButton';
+import { ICON_SIZES, ICONS } from '../Icon';
+import DsButton, { BUTTON_COLORS, BUTTON_TYPES } from '../Button';
+import DsModal from '../Modal';
+import SurveyToggle, {
+	SURVEY_TOGGLE_MEANINGS,
+	SURVEY_TOGGLE_STATES,
+	SURVEY_TOGGLE_STATUSES,
+} from '../SurveyToggle';
+import { SURVEY_QUESTION_STATES } from './SurveyQuestion.consts';
+import SurveyQuestionTextarea from './SurveyQuestionTextarea.vue';
+import { SurveyQuestionScaleOption } from './SurveyQuestion.domain';
+import { randomString } from '../../utils/string';
+
+export default {
+	name: 'SurveyQuestionScale',
+	components: {
+		SurveyQuestionTextarea,
+		DsCard,
+		IconButton,
+		SurveyToggle,
+		DsModal,
+		DsButton,
+	},
+	props: {
+		title: {
+			type: String,
+			required: true,
+		},
+		state: {
+			type: String,
+			default: SURVEY_QUESTION_STATES.DEFAULT,
+			validate(state) {
+				return Object.values(SURVEY_QUESTION_STATES).includes(state);
+			},
+		},
+		scaleOptions: {
+			type: Array as Prop<Array<SurveyQuestionScaleOption>>,
+			required: true,
+			validate(scaleOptions) {
+				return scaleOptions.every((option) => typeof option === 'object');
+			},
+		},
+		elaborationLabel: {
+			type: String,
+			required: true,
+		},
+		elaborationValue: {
+			type: String,
+			required: true,
+		},
+		placeholder: {
+			type: String,
+			default: 'Wpisz swoją odpowiedź',
+		},
+		selectedValue: {
+			type: String,
+			default: null,
+		},
+	},
+	data() {
+		return {
+			showModal: false,
+			inputId: 'survey-question-' + randomString(8),
+		};
+	},
+	created() {
+		this.BUTTON_COLORS = BUTTON_COLORS;
+		this.BUTTON_TYPES = BUTTON_TYPES;
+		this.ICONS = ICONS;
+		this.ICON_SIZES = ICON_SIZES;
+		this.SURVEY_TOGGLE_COLORS = SURVEY_TOGGLE_MEANINGS;
+		this.SURVEY_TOGGLE_STATES = SURVEY_TOGGLE_STATES;
+		this.SURVEY_TOGGLE_STATUSES = SURVEY_TOGGLE_STATUSES;
+		this.SURVEY_QUESTION_STATES = SURVEY_QUESTION_STATES;
+	},
+	methods: {
+		onToggleClick(value: number) {
```

Comment:
@kvas-damian I think it should be `: string`

Reply:
https://github.com/bethinkpl/design-system/commit/f8f1bdacfafa909f7a75dccf3f1db7b67fb4bbb2